### ISSUE TITLE
Field has no label or title attribute

### ISSUE THREAD

**Issue by greenc05** [bbc/bbc-a11y#94]
When running the BBC-a11y tool, the following error is found with the locator submit input on http://bbc.co.uk/news

Field has no label or title attribute: /html/body/div[1]/div[6]/div/div[4]/div[2]/div[2]/div[1]/div[1]/div[16]/div/div[1]/div/form/div/input

Should this code be valid @EmmaJP? 

<input id="locator-submit" name="locator-submit" class="submit" type="submit" value="Search">

According W3C https://www.w3.org/TR/WCAG20-TECHS/H65.html in example number 3 we don't need a label or title attribute in this case as there is a value attribute, please could you confirm @EmmaJP?

**Comment by EmmaJP** [comment]
It does seem that is correct, http://webaim.org/techniques/forms/controls#button, so perhaps we need to adjust the guideline and tests specifically in regard to submit, reset, and button elements.
